subsys/net/conn_mgr/Kconfig

```diff
@@ -36,4 +36,8 @@ config NET_CONNECTION_MANAGER_PRIORITY
 	help
 	  This sets the starting priority of the connection manager thread.
 
+config NET_CONNECTION_MANAGER_AUTO_IF_DOWN
+	bool "Automatically take ifaces admin-down if they have stopped trying to connect"
```

Comment:
I am a bit baffled what the `they have stopped` refers to, do you mean application has stopped trying to connect or what? Could this text clarified or help text added to this one?

Reply:
`they` in this case refers to `ifaces`

Perhaps the following would be clearer?

`Automatically call net_if_down on ifaces that have given up on connecting`